Cython/Compiler/ModuleNode.py

```diff
@@ -233,8 +233,29 @@ def h_entries(entries, api=0, pxd=0):
                 h_code.putln('#error "Unicode module names are not supported in Python 2";')
             h_code.putln("PyMODINIT_FUNC init%s(void);" % py2_mod_name)
             h_code.putln("#else")
-            h_code.putln("PyMODINIT_FUNC %s(void);" % self.mod_init_func_cname('PyInit', env))
-            h_code.putln("#endif")
+            py3_mod_func_name = self.mod_init_func_cname('PyInit', env)
+            warning_string = EncodedString(
+                'Use PyImport_AppendInittab("%s", %s) instead of calling %s directly.' % (
+                                    py2_mod_name, py3_mod_func_name, py3_mod_func_name))
```

Comment:
I noticed that you are using three different levels of indentation for the string formatting arguments across just 20 lines of code. :) It would be nice if we could keep that to just one level (4 spaces) overall.